__init__.py

```diff
@@ -25,10 +25,35 @@
 
 FIFTEEN_MINUTES = 900
 MARK_II = "mycroft_mark_2"
+ONE_HOUR = 3600
 ONE_MINUTE = 60
 TEN_SECONDS = 10
 
 
+def _find_latest_skill_update(skill_info: dict) -> str:
+    """Returns a string representation of the last time a skill was updated.
+
+    Args:
+        skill_info: contents of the installed skill data file built by MSM
+    """
+    latest_update = 0
+    for skill_attributes in skill_info["skills"]:
+        installed_timestamp = skill_attributes["installed"]
+        updated_timestamp = skill_attributes["updated"]
+        if installed_timestamp > latest_update:
+            latest_update = installed_timestamp
+        if updated_timestamp > latest_update:
+            latest_update = updated_timestamp
+
+    if latest_update:
+        latest_update_datetime = datetime.fromtimestamp(latest_update)
+        skill_update_datetime = latest_update_datetime.strftime("%Y-%m-%d %H:%M")
```

Comment:
Would it be strange to have the build timestamp in UTC but the Skills timestamp in the devices local timezone? 

I just went to check when my next Skill update would be, got confused, then realised it's in UTC and so need to do a calculation to work out how that time relates to the current time shown on the screen.

Compared to the Build timestamp where the key is having a single consistent value for everyone regardless of where they are on the planet.

Reply:
So you are recommending the skill date/time be in local time?

Reply:
As discussed in the dev sync meeting, this needs to be UTC for now so we can compare dates across devices to know if we are running the same version of skill code